scripts/linters/pylint_extensions_test.py

```diff
@@ -335,7 +335,7 @@ class ClassName(dummy_class):
         node_no_newline_below_class_docstring.file = filename
         node_no_newline_below_class_docstring.path = filename
 
-        self.checker_test_object.checker.visit_classdef(
```

Comment:
If the tests were wrongly written you should fix them and test the code, not rename the check. Also you said the actual code would be passing Class node not module node, then why doesn't the current checker catches those?